content/applications/marketing/events/create_events.rst

```diff
@@ -0,0 +1,402 @@
+
+.. tip::
+   Existing event registration products can be modified directly from this field, as well, by clicking
+   the :guilabel:`Internal link ➡️ (right arrow)` icon, located beside the event registration product.
+   Doing so reveals that product's page on a separate page. Then, proceed to make any desired
+   modifications.
+
+Add or modify the cost of the ticket in the :guilabel:`Price` field.
+
+.. note::
+   It should be noted that a modified ticket price on the event form, in the :guilabel:`Tickets`
+   tab, does **not** change the price that appears on the ticket's product form. This modification
+   is specific to the event being configured.
+
+Next, determine a :guilabel:`Sales Start` and :guilabel:`Sales End` date in their respective fields.
+To do that, click into the blank field to reveal a calendar popover. From there, select the desired
+date and time, then click :guilabel:`✔️ Apply`.
+
+Then, if desired, designate a :guilabel:`Maximum` amount of that specific ticket that can be sold.
+
+The :guilabel:`Taken` column will populate with accurate data as tickets are sold.
+
+In the :guilabel:`Color` column, users have the option to add a custom color code to different
+tickets.
+
+If the user wishes to delete any tickets from the :guilabel:`Tickets` tab, click the :guilabel:`🗑️
+(trash can)` icon on the corresponding line for the ticket that should be deleted.
+
+.. tip::
+   To add an optional :guilabel:`Description` column to the :guilabel:`Tickets` tab, click the
+   :guilabel:`(optional columns drop-down)` menu, located to the far-right of the column titles.
+   Then, tick the checkbox beside :guilabel:`Description` from the resulting drop-down menu.
+
+   When added, users have the option to add brief descriptions for each event ticket created, which
+   can be used to inform attendees of any perks or amenities that may coincide with specific tickets.
+
+.. _create_events/event_form/communication:
+
+Communication tab
+-----------------
+
+In the :guilabel:`Communication` tab of an event form, users can create various communications that
+can be scheduled to be sent at specific intervals leading up to, and following, the event.
+
+.. image:: create_events/communication-tab.png
+   :align: center
+   :alt: Typical communication tab on an event form in the Odoo Events application.
+
+.. note::
+   By default, Odoo provides three separate pre-configured communications on every new event form.
+   One is an email sent after each registration to confirm the purchase with the attendee. The other
+   two are email event reminders that are scheduled to be sent at different time intervals leading
+   up to the event to remind the recipient of the upcoming event.
+
+To add a communication in the :guilabel:`Communication` tab, click :guilabel:`Add a line`. Then,
+select the desired type of communication in the :guilabel:`Send` field. The options are:
+:guilabel:`Mail`, :guilabel:`SMS`, :guilabel:`Social Post`, or :guilabel:`WhatsApp`.
```

Comment:
Doc-wide observation: I'd be careful about sandwiching small paragraphs like this in between admonition blocks, as it's very easy to miss them! Consider clumping these with other body paragraphs or condense similar admonition blocks together, where appropriate to do so.